Added examples and definition - CSS

chef-danny-d opened the pull request:
- added two examples with the most used values
- added definition

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

Commit:
Added examples and definition - CSS

- added two examples with the most used values
- added definition

Commit:
Added resources to Graceful Degradation - CSS

Added links to multiple articles / post / explanations / tools to help coders understand the concept

b3u commented:
LGTM
BTW, use one backtick instead of three for inline code. "(```initial``` vs. `initial`)

Commit:
changed triple backtick to single

chef-danny-d commented:
@b3u appreceat it!!! changed it to single ticks.

huyenltnguyen commented on guide/english/css/graceful-degradation/index.md:
```diff
@@ -17,4 +17,18 @@ title: Graceful Degradation
+
+[W3C CSS validator](http://jigsaw.w3.org/css-validator/)
+
+[Online CSS linter](http://csslint.net/)
```
This website is not supporting `https`, which is not conform with the [freeCodeCamp's Style Guide](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/style-guide-for-guide-articles.md#links). Please remove it from your changes.

Commit:
fixed https link

- had to fix one link to point to the https site
- removed one link because it doesn't match with the guide provided by FCC

huyenltnguyen committed:
Add a space between class name and curly brace

huyenltnguyen commented:
Hi @chef-danny-d, thank you for the PR!

I really appreciate your effort to improve the article. I'm merging your PR now.

Congratulations on your first contribution to this repo ✨ 🎉 We look forward to your next.

Happy coding!
